scripts/analysis.py

```diff
@@ -248,114 +248,6 @@ def timeseries_cum(specific_search, duration, kg_to_tons):
     return value_timeseries
 
 
-def get_new_deployment(power_dict, inst_list, demand_eq, new_reactor_power,
```

Comment:
why delete this section of this file, but not the whole file?

Reply:
The two functions `get_new_deployment` and `write_deployinst` were placed wrongly (by me, of course). Analysis.py should handle backend analysis but these two functions help write reactor deployment schemes. I added the two functions as a submodule in `cyclus_input_gen` (with tests)